cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/IOUContract.kt

```diff
@@ -49,5 +64,6 @@ class IOUContract : Contract {
      */
     interface Commands : CommandData {
         class Create : Commands
+        class Destroy : Commands
```

Comment:
Where are the tests for this new feature?

Reply:
II thought tests are later part of the course and I didn't do it. I will add some :)

Reply:
Fair enough but contract tests come before any flow anyway.